tests/jsmarionette/runner/marionette-js-runner/host/lib/request.js

```diff
@@ -71,6 +77,13 @@ module.exports = function request(socketPath, path, json, retry) {
           err.stack = json.stack;
           reject(err);
         } else {
+          // start_runner is special and troublesome.
```

Comment:
Could you describe a bit more what's happening when we get a `/start_runner` response without having received any data?